explore_warming.ipynb

```diff
@@ -0,0 +1,746 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'climakitae'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
+      "\u001b[0;32m/tmp/ipykernel_348/2482252592.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0mpn\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mextension\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0;32mimport\u001b[0m \u001b[0mclimakitae\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mck\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
+      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'climakitae'"
+     ]
+    }
```

Comment:
Can you clear the output of the cells in this notebook so this error message is not shown? 


<img width="800" alt="error" src="https://user-images.githubusercontent.com/66140951/199822913-3c4dad02-8b51-4176-86d4-0740b1057d7a.png">